queries.txt

```diff
@@ -0,0 +1,71 @@
+1. select * 
+from subjects 
+order by subject asc;
+
+2. select * 
+from subjects 
+order by location asc;
+
+3. select * 
+from books 
+where title ='Little Women';
+
+4. select * 
+from books 
+where title ilike '%python%';
+
```

Comment:
Good, you could also use like - select * from books where title like '%Python%';